redi-connect-backend/server/server.js

```diff
@@ -77,7 +77,7 @@ app.start = function () {
         cert: sslConfig.certificate
       }, app)
     } else {
-      return http.crates.createServer(app)
```

Comment:
is this a typo? While looking up the hotfix you merged @ericbolikowski and comparing it with the older version seemed to me this `crates` shouldnt be there (?)
https://github.com/talent-connect/connect/commit/56b0fa46065dfdf3e56bc52a89ec76b11458b999#diff-4b9d30ab8b5e8400e157162a189007c5adee7015f252ffacef77274b5c95847fL74